manifests/t/TortoiseSVN/TortoiseSVN/1.14.3.29387/TortoiseSVN.TortoiseSVN.locale.en-US.yaml

```
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.5-1-22598-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: TortoiseSVN.TortoiseSVN
+PackageVersion: 1.14.3.29387
+PackageLocale: en-US
+Publisher: TortoiseSVN
+# PublisherUrl: 
+PublisherSupportUrl: https://tortoisesvn.net/support.html
+# PrivacyUrl: 
+# Author: 
+PackageName: TortoiseSVN
+PackageUrl: https://tortoisesvn.net
+License: GPL
```

```suggestion
License: GPL-2.0-only
```
https://osdn.net/projects/tortoisesvn/scm/svn/blobs/head/branches/1.14.x/src/gpl.txt